docs/source/_toctree.yml

```diff
@@ -22,6 +22,8 @@
       title: Overview
     - local: guides/setup_aws_instance
       title: Set up AWS Trainium instance
+    - local: guides/sagemaker
+      title: Using Amazon SageMaker
```

Comment:
could you point out the objective in the title? Things like `Training and deployment using Amazon Sagemaker`?

Reply:
good idea